src/Rule/NoBracketsInMethodDirective.php

```diff
@@ -13,22 +13,18 @@
 
 namespace App\Rule;
 
-use App\Annotations\Rule\Description;
-use App\Annotations\Rule\InvalidExample;
-use App\Annotations\Rule\ValidExample;
+use App\Attribute\Rule\Description;
+use App\Attribute\Rule\InvalidExample;
+use App\Attribute\Rule\ValidExample;
 use App\Value\Lines;
 use App\Value\NullViolation;
 use App\Value\RuleGroup;
 use App\Value\Violation;
 use App\Value\ViolationInterface;
 
-/**
- * @Description("Ensure a :method: directive has special format.")
- *
- * @InvalidExample(":method:`Symfony\\Component\\OptionsResolver\\Options::offsetGet()`")
- *
- * @ValidExample(":method:`Symfony\\Component\\OptionsResolver\\Options::offsetGet`")
- */
+#[Description('Ensure a :method: directive has special format.')]
+#[InvalidExample(':method:`Symfony\\\\Component\\\\OptionsResolver\\\\Options::offsetGet()`')]
```

Comment:
Do you know how we can avoid `\\\\` ?

Else rendering is not the same

Reply:
Sorry, I don't understand the problem 🤔